pending/Driver Leftovers.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    BleachBit
+    Copyright (C) 2008-2019 Andrew Ziem
+    https://www.bleachbit.org
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<cleaner id="Driver Leftovers" os="windows">
+  <label>Driver Leftovers</label>
+  <description>Deletes staging folders with driver installer data.</description>
+  <option id="NVIDIA">
+    <label>NVIDIA</label>
+    <description>Deletes C:\Nvidia [Used for Nvidia Drivers]</description>
+    <action search="walk.top" command="delete" path="C:\Nvidia"/>
+  </option>
+  <option id="AMD">
+    <label>AMD</label>
+    <description>Deletes C:\AMD [used for AMD drivers]</description>
+    <action search="walk.top" command="delete" path="C:\AMD"/>
+  </option>
+  <option id="Temp">
+    <label>Temp</label>
+    <description>Deletes C:\Temp [Used for numerous applications, including RTX Voice]</description>
+    <action search="walk.top" command="delete" path="C:\Temp"/>

```

Comment:
As you mentioned, `c:\temp\` is used by many applications. They may not be related at all to "driver leftovers." Could you please make this action more specific to "driver leftovers"?

Reply:
The only application I had this year was RTX Voice, regardless it's still temporary, usually an installer staging folder